www/src/pages/foundations/spacing.tsx

```diff
@@ -30,21 +32,23 @@ export type PixelCellTypes = {
   spacer: number,
 };
 
-const PixelCell = ({ spacer }: PixelCellTypes) => (
-  <MeasuredItem
-    properties={['margin']}
-    renderAfter={(measurements: { margin: number }) => (
-      <code>
-        {measurements.margin}
-      </code>
-    )}
-  >
-    <div
-      style={{ display: 'none' }}
-      className={`m-${spacer}`}
-    />
-  </MeasuredItem>
-);
+function PixelCell({ spacer }: PixelCellTypes) {
```

Comment:
This refactoring is related to: ESLint: Function component is not a function declaration(react/function-component-definition)